data/presets/shop/hairdresser.json

```diff
@@ -5,7 +5,9 @@
         "area"
     ],
     "terms": [
-        "barber"
+        "barber",
+        "beard",
+        "salon"
```

Comment:
I'm still not clear on exactly how iD uses `terms` vs `aliases`, but it sounds like `terms` is supposed to be for related terms and `aliases` is for synonyms. If that is correct, then I'd say `barber` and `salon` could be aliases.  Might also add `haircut` to terms.

Reply:
“Hair Salon” and “Barber Shop” would be good aliases. Neither term is quite gender-neutral, but that shouldn’t be a problem for aliases, right?

Reply:
Oh yes, ~barber~ Barber Shop and ~salon~ Hair Salon.  Better.

Reply:
Not sure if we need to wait on https://github.com/ideditor/schema-builder/pull/57 then. Is a barber shop a real synonym or maybe "just" a subtype? (But "Hair Salon" I can add)